README.md

```diff
@@ -7,7 +7,10 @@ Logux is a new way to connect client and server. Instead of sending
 HTTP requests (e.g., AJAX and GraphQL) it synchronizes log of operations
 between client, server, and other clients.
 
-**Documentation: [logux.io]**
+* **[Guide and recipes](https://logux.io/)**
+* **[Chat](https://gitter.im/logux/logux)** for any questions
+* **[Issues and plans](https://github.com/logux/logux/issues)**
+* **[Projects](https://logux.io/architecture/parts/)** inside the Logux family
```

Comment:
Since Logux has Django, Vuex, Redux and other, it can be called ecosystem.

Reply:
We can also rename this page of the site, add a link to the chat and other parts of ecosystem in the future.

Reply:
Fixed https://github.com/logux/vuex/pull/28/commits/a491e7d46cda4564b7827fa38b5d6ccf7ec543ee